File: Convert from HTML to PDF in CSharp VB and ASP.NET with a Free 3rd Party Library/[C#]-Convert from HTML to PDF in CSharp VB and ASP.NET with a Free 3rd Party Library/C#/SelectPdf.Samples/Samples.Master.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;
using ColorCode;

namespace SelectPdf.Samples
{
    public partial class Samples : System.Web.UI.MasterPage
    {
        protected void Page_Load(object sender, EventArgs e)
        {
            if (Request.Url.AbsolutePath.ToLower().Contains("default.aspx"))
            {
                LitCode.Visible = false;
            }
            else
            {
                try
                {
                    LitCode.Text = @"<h1>Sample Code C#</h1><br/><br/>";
                    string code = System.IO.File.ReadAllText(Server.MapPath(Request.Url.AbsolutePath) + ".cs");
                    LitCode.Text += "<div style='width=90%; overflow: auto; border: 1px solid #DDDDDD; padding: 10px;'>" + new CodeColorizer().Colorize(code, Languages.CSharp) + "</div>";
                }
                catch
                {
                    LitCode.Visible = false;
                }
            }
        }
    }
}

File: Convert from HTML to PDF in CSharp VB and ASP.NET with a Free 3rd Party Library/[VB]-Convert from HTML to PDF in CSharp VB and ASP.NET with a Free 3rd Party Library/VB/SelectPdf.Samples/Samples.Master.vb
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports ColorCode

Namespace SelectPdf.Samples
	Public Partial Class Samples
		Inherits System.Web.UI.MasterPage

        Protected Sub Page_Load(sender As Object, e As EventArgs)
            If Request.Url.AbsolutePath.ToLower().Contains("default.aspx") Then
                LitCode.Visible = False
            Else
                Try
                    LitCode.Text = "<h1>Sample Code Vb.Net</h1><br/><br/>"
                    Dim code As String = System.IO.File.ReadAllText(Server.MapPath(Request.Url.AbsolutePath) + ".vb")
                    LitCode.Text = LitCode.Text & "<div style='width=90%; overflow: auto; border: 1px solid #DDDDDD; padding: 10px;'>" & New CodeColorizer().Colorize(code, Languages.VbDotNet) & "</div>"
                Catch
                    LitCode.Visible = False
                End Try
            End If
        End Sub
	End Class
End Namespace
